docker-compose.yml

```diff
@@ -19,6 +20,15 @@ web:
     - MYSQL_ROOT_PASSWORD='docker'
     - MYSQL_DATABASE='olympia'
     - PYTHONDONTWRITEBYTECODE=1
+    - BROKER_URL=amqp://olympia:olympia@rabbitmq/olympia
+
+worker:
+  build: .
```

Comment:
`extends` doesn't work with services that have `links`. So using `extends` means that we build some generic `olympia` service that does nothing before building the other two and does not seem to give any benefit. The three builds (`olympia`, `worker` and `web`) all take about the same length of time.